core/src/test/scala/org/apache/spark/ui/storage/StorageTabSuite.scala

```diff
@@ -179,6 +179,25 @@ class StorageTabSuite extends SparkFunSuite with BeforeAndAfter {
     assert(storageListener.rddInfoList.size === 2)
   }
 
+  test("verify StorageTab still contains a renamed RDD") {
+    {
+      val rddInfo = new RDDInfo(0, "original_name", 1, memOnly, Seq(4))
+      val stageInfo0 = new StageInfo(0, 0, "stage0", 1, Seq(rddInfo), Seq.empty, "details")
+      bus.postToAll(SparkListenerBlockManagerAdded(1L, bm1, 1000L))
+      bus.postToAll(SparkListenerStageSubmitted(stageInfo0))
+      val blockUpdateInfos1 = Seq(BlockUpdatedInfo(bm1, RDDBlockId(0, 1), memOnly, 100L, 0L))
+      postUpdateBlocks(bus, blockUpdateInfos1)
+      assert(storageListener.rddInfoList.size == 1)
+    }
```

Comment:
Nope, it's only there to increase readability (to limit the scope of `rddInfo`, `stageInfo0` and `blockUpdateInfos1` which aren't needed later in the test).